app/src/main/java/com/github/sdpteam15/polyevents/view/PolyEventsApplication.kt

```diff
@@ -10,11 +10,21 @@ import com.github.sdpteam15.polyevents.R
 import com.github.sdpteam15.polyevents.model.database.local.room.LocalDatabase
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.launch
 
 // TODO: consider instantiating Firebase database here
 class PolyEventsApplication : Application() {
     companion object{
         var inTest = false
+        lateinit var application :PolyEventsApplication
+    }
+    override fun onCreate() {
+        super.onCreate()
+        application = this
+
+        applicationScope.launch {
+            database.genericEntityDao().deleteAll()
+        }
```

Comment:
Why do you remove all entity from the database at Application Launch ?

Reply:
oh, it was to test if it work well and I forget to remove it.
```suggestion
        lateinit var application :PolyEventsApplication
    }
    override fun onCreate() {
        super.onCreate()
        application = this
```